src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -1837,10 +1833,12 @@ def aks_agentpool_update(cmd,   # pylint: disable=unused-argument
         disable_cluster_autoscaler + update_cluster_autoscaler
 
     if (update_autoscaler != 1 and not tags and not scale_down_mode and not mode and not max_surge and labels is None and node_taints is None):
-        raise CLIError('Please specify one or more of "--enable-cluster-autoscaler" or '
-                       '"--disable-cluster-autoscaler" or '
-                       '"--update-cluster-autoscaler" or '
-                       '"--tags" or "--mode" or "--max-surge" or "--scale-down-mode" or "--labels" or "--node-taints')
+        reconcilePrompt = "no argument specified to update would you like to reconcile to current settings?"
```

Comment:
minor: Use `'` rather than `"` for consistency, since from what I can tell that's the style used elsewhere in this file.